dev/com.ibm.ws.classloading_sharedclass_fat/test-resourceadapters/sharedClassesRar/resources/META-INF/ra.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<connector xmlns="http://java.sun.com/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.6" xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/connector_1_6.xsd">
```

Comment:
Since you are testing with Java / Jakarta EE 8, you might as well use the 1.7 version of the schema.